src/server.ts

```diff
@@ -4,10 +4,10 @@ import { createServer } from 'http'
 import compression from 'compression'
 import cors from 'cors'
 
-import findOrCreateProfile from './findOrCreateProfile'
-
+// Load the env vars before initializing code that depends on them
 dotenv.config()
 
+import findOrCreateProfile from './findOrCreateProfile'
```

Comment:
The order matters here if we wanna skip auth

Reply:
🤦🏼‍♂️